Trappist/src/Promact.Trappist.Test/Questions/QuestionsRepositoryTest.cs

```diff
@@ -357,6 +360,60 @@ public QuestionsRepositoryTest(Bootstrap bootstrap) : base(bootstrap)
         }
 
         /// <summary>
+        /// Method to test IsQuestionExistInTestAsync() method
+        /// </summary>
+        [Fact]
+        public async Task IsQuestionExistInTestTest()
```

Comment:
what if the question is not exists in the test? that scenario needs to mention in the test case.